Lib/pydoc.py

```diff
@@ -567,49 +567,46 @@ class HTMLDoc(Doc):
     def page(self, title, contents):
         """Format an HTML page."""
         return '''\
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
-<html><head><title>Python: %s</title>
+<!DOCTYPE html>
+<html lang="en"><head><title>Python: %s</title>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
```

Comment:
```suggestion
<meta charset=utf-8>
```

I think it should also be before the title; you need to know the character encoding to know how to decode the title.

Reply:
Thanks, fixed.